doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+      parameters:
+        - name: start_height
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: List of blocks
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Block'
+  /block-height/{height}:
+    get:
+      tags:
+        - block
+      summary: Get block header by height
+      parameters:
+        - name: height
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Block header
+          content:
+            application/json:
+              schema:
+                type: string
+  /block/{hash}:
+    get:
+      tags:
+        - block
+      summary: Returns information about a block
+      description: |
+          Elements-based chains have an additional `proof` field. <!--TODO: add link to block-format -->
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Block data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Block'
+  /block/{hash}/status:
+    get:
+      tags:
+        - block
+      summary: Get block status by hash
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Block status
+          content:
+            application/json:
+              schema:
+                type: object
+  /block/{hash}/txids:
+    get:
+      tags:
+        - block
+      summary: Returns a list of Transaction ID's in a block.
+      description: Responses for this endpoint can be cached indefinitely.
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Transaction IDs
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+  /block/{hash}/header:
+    get:
+      tags:
+        - block
+      summary: Returns the Hex Encoded block header
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Block header
+          content:
+            text/plain:
+              schema:
+                type: string
+  /block/{hash}/raw:
+    get:
+      tags:
+        - block
+      summary: Returns the binary respresentation of the block
+      description: Responses can be cahced indefinitely.
```

Comment:
typo cahced

Reply:
fixed spelling and moved to to response description.